poup.css

```diff
@@ -0,0 +1,233 @@
+.popup_window {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  display: none;
+  top: 1%;
+  bottom: 0;
+  overflow: auto;
+}
+
+.popup_skills_mob li {
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.36px;
+  background-color: #ebecf0;
+  color: #344563;
+  padding: 10px 10px;
+  border-radius: 5px;
+  margin-right: 1%;
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 13px;
+  justify-content: center;
+  width: 80%;
+  padding: 75px 0;
+  margin-left: 8%;
+  background-color: #fff;
+}
+
+.popup .mob_snap {
+  width: 90%;
+  text-align: center;
+  align-items: center;
+
+  /* display: flex;
+   height: 220px;
+  padding: 8px 8px 172px 263px;
+  justify-content: flex-end;
+  align-items: center;
+  border-radius: 8px; */
+}
+
+.popup_btn_src img {
+  color: #fff;
+  width: 18px;
+  height: 20px;
+  flex-shrink: 0;
+}
+
+.popup_btn_live img {
+  color: #fff;
+  width: 24px;
+  height: 24px;
+}
+
+.popup .desktop_snap {
+  display: none;
+}
+
+.popup_heading h3 {
+  display: flex;
+  flex-direction: column;
+  color: #172b4d;
+  font-size: 32px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 44px;
+}
+
+.popup_heading {
+  width: 60vw;
+  margin-top: 20px;
+}
+
+.popup_skills_desktop {
+  display: none;
+}
+
+.popup_skills_mob {
+  display: flex;
+  margin: 15px 0;
+  color: #3a4a42;
+  font-size: 12px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.36px;
+}
+
+.popup-page {
+  width: 80%;
+  margin-top: 22px;
+}
+
+/* .popup_skills {
+  display: flex;
+  flex-direction: column;
+  background: white;
+  width: 375px;
+  height: 882px;
+  margin: 12px 0;
+} */
+
+.popup_skills ul {
+  list-style: none;
+}
+
+.popup_description_Mob p {
+  margin: 12px 0;
+}
+
+.popup_description_Mob {
+  font-size: 16px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.popup_description_Desktop {
+  display: none;
+}
+
+.popup_btn {
+  display: flex;
+  width: 80%;
+  justify-content: space-between;
+
+  /*  align-items: flex-start;
+  border-radius: 4px;
+  gap: 12px;
+  background: #36b37f; */
+}
+
+.popup_btn button {
+  display: flex;
+  align-items: center;
+  padding: 12px 16px;
+  font-size: 1em;
+  gap: 5px;
+}
+
+.popup_btn button a {
+  color: #fff;
+}
+
+.close {
+  /* width: 24px;
+  height: 24px; */
```

Comment:
- [ ] Please 🙏🏻 remove these line of code from your `file` because having commented code in the codebase makes it cluttered, confuses developers, and can create maintenance and version control issues. Kindly fix it and it should be avoided for clean, professional and standardized code.